cme/lms/templates/student_account/register-extra.html

```diff
@@ -0,0 +1,12 @@
+<%!
+    from django.utils.translation import ugettext as _
+%>
+<fieldset>
+    <legend>&dagger; Medical Designation or Professional Title</legend>
```

Comment:
This is how it was previously, but should we just go ahead and i18n this?

Reply:
I think we want it exactly as written, so no i18n...

Reply:
It's not `i18n`'ed elsewhere, so that's fine.